tests/integration/test_notebooks.py

```diff
@@ -147,21 +151,35 @@ def test_movielens(asv_db, bench_info, tmpdir):
     os.environ["MODEL_BASE_DIR"] = INFERENCE_MULTI_HOT
     os.environ["MODEL_PATH"] = INFERENCE_MULTI_HOT
 
+    # Run HugeCTR
+    try:
+        notebook = os.path.join(
+            dirname(TEST_PATH), MOVIELENS_DIR, "inference-HugeCTR/Training-with-HugeCTR.ipynb"
+        )
+        import hugectr
+
+        print(hugectr.__version__)
+        # _run_notebook(tmpdir, notebook, data_path, input_path, gpu_id=devices, clean_up=False)
+        return
+    except ImportError:
+        print("HugeCTR not installed, skipping " + notebook)
+
+    # Run Tensorflow or PyTorch
```

Comment:
doing inference before preprocessing and training?

Reply:
That notebook actually has Download. preproc, and ETL. It does not follow the TF and PyTorch format; However, it is being updated this release; so I will remove it from this PR (That is one of the reasons it was commented) and I will addut following the standard format when it is ready